pkg/network/vmispec/interface.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package vmispec
+
+import (
+	v1 "kubevirt.io/api/core/v1"
+)
+
+func FilterSRIOVInterfaces(ifaces []v1.Interface) []v1.Interface {
+	var sriovIfaces []v1.Interface
+	for _, iface := range ifaces {
+		if iface.SRIOV != nil {
+			sriovIfaces = append(sriovIfaces, iface)
+		}
+	}
+	return sriovIfaces
+}
+
+func PodNetworkHasOnlyMasqueradeBindingInterfaces(networks []v1.Network, ifaces []v1.Interface) bool {
```

Comment:
AFAIK, it is not possible to have more than one interface pointing to a single pod network. Therefore, this method confusing.

Reply:
Done